File: MyTest/MyTest/src/com/ds/BubbleSort.java
package com.ds;

public class BubbleSort {

	public static void main(String[] args) {
		int[] arr = { 30, 10, 50, 20, 60, 40 };
		bubbleSort(arr);
	}

	private static void bubbleSort(int[] arr) {
		int n = arr.length;
		for (int i = 0; i < n-1; i++) {
			for (int j = 0; j < n - i - 1; j++) {
				if (arr[j + 1] < arr[j]) {
					int temp = arr[j + 1];
					arr[j + 1] = arr[j];
					arr[j] = temp;
				}
			}

		}

		for (int i : arr) {
			System.out.print(i + " ");
		}

	}

}

File: dspreparation/bubbleSort.js
function bubbleSort(arr){
  for(let i =0;i< arr.length;i++){
    for(let j = 0 ; j< arr.length;j++){
      if(arr[j+1] < arr[j]){
        let temp = arr[j+1];
        arr[j+1] = arr[j];
        arr[j] = temp;
      }
    }
  }
  
  console.log(arr);
}

let arr = [8,5,9,10,2,1];
bubbleSort(arr);